source/pacboy/pacboy.sh

```diff
@@ -13,20 +13,14 @@ help_and_exit() {
         $(basename "$0") [command] [arguments]
         Arguments will be passed to pacman after translation:
 
-        For 64-bit MSYS2 shell:
+        Package name translations:
             name:i means i686-only
             name:x means x86_64-only
             name:z means clang-i686-only
             name:c means clang-x86_64-only
             name:u means ucrt-x86_64-only
             name:a means clang-aarch64-only
             name:p means MINGW_PACKAGE_PREFIX-only
```

Comment:
Do I understand correctly that `name` will be the same as `name:p`? Would be better to state explicitly what `name` does in the help.

Reply:
good point, I guess that is indeed the case now

Reply:
Maybe the other way around?
```
name - uses the package prefix of the current environment: ${full_package_prefix}name
name:p - means the same as \"name\"
```

Reply:
ok, done